requirements.txt

```diff
@@ -0,0 +1,11 @@
+cycler==0.10.0
+joblib==0.14.1
+kiwisolver==1.1.0
+numpy==1.18.1
+pkg-resources==0.0.0
+pyparsing==2.4.6
+python-dateutil==2.8.1
+scikit-learn==0.22.1
+scipy==1.4.1
+six==1.14.0
+tqdm==4.43.0
```

Comment:
Are you sure all these packages are necessary? I would prefer to have the minimal version if possible. Or maybe we could make two requirements versions, one complete and one minimal.